policy/config/lsstFamNoPertPointer.yaml

```diff
@@ -7,6 +7,6 @@ input:
 gal: '{TS_IMSIM_DIR}/policy/config/gal/starDefault.yaml'
 image: '{TS_IMSIM_DIR}/policy/config/image/lsstCamDefault.yaml'
 psf: '{TS_IMSIM_DIR}/policy/config/psf/psfDefault.yaml'
-stamp: '{TS_IMSIM_DIR}/policy/config/stamp/lsstCamDefault.yaml'
-output: '{TS_IMSIM_DIR}/policy/config/output/lsstFamDefault.yaml'
+stamp: '{TS_IMSIM_DIR}/policy/config/stamp/lsstCamLsstSiliconWithDiffraction.yaml'
+output: '{TS_IMSIM_DIR}/policy/config/output/lsstFamRaftCenters.yaml'
```

Comment:
/Would it make sense to have a yaml that is `lsstCamNoDiffractionPointer`?

Reply:
We could but the design here is that users can make whatever pointers they want. These are just some samples that are our defaults for testing. Based upon the work I've shown in the related JIRA issue and talking to Josh we are going to move to including diffraction as the default.